packages/cli/test/ci-collect.test.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
let's just name this file `collect.test.js`

Reply:
Done.